splunk/src/main/java/com/splunk/Receiver.java

```diff
@@ -83,8 +83,7 @@ public Socket attach(Args args) throws IOException {
      */
     public Socket attach(String indexName, Args args) throws IOException {
         Socket socket = service.open();
-        OutputStream ostream = socket.getOutputStream();
-        Writer out = new OutputStreamWriter(ostream, "UTF-8");
+        PrintWriter writer = new PrintWriter(new OutputStreamWriter(socket.getOutputStream(), "UTF-8"));
```

Comment:
Why is the type of writer being changed here? It's not clear from the PR summary.

Reply:
PR summary updated.